src/visualMarkers/main.ts

```diff
@@ -73,6 +86,15 @@ async function handleMarkChangeEvent(event: MarkChangeEvent) {
 
       await signalFlare(addedMark.position, color, azimuth)
 
+      // if duration is set by the user, repeat flaring in interval
+      if (command.duration != undefined) {
+        const timerID = setInterval(
+          async () => await signalFlare(addedMark.position, color, azimuth),
```

Comment:
since this is identical to the above, we should probably make this a pattern that we can re-use in both places instead of copy pasting the whole implementation